styleguide/source/_patterns/01-molecules/09-search-results/search-result-list-external.twig

```diff
@@ -0,0 +1,27 @@
+{#
+ # @file search-result-list-external.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="search-result_list-external">
+  <div class="layout layout_one_up">
+    {% include 'atoms-rule-horizontal' %}
+  </div>
+  <div class="layout layout_three_up_heavy_center">
```

Comment:
This should probably be a block level link, based on the design hover state for the comps